SunCalcNet.Tests/SunCalcNet.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
```

Comment:
test project could target only net7.0

Reply:
Thanks for this feedback, I have removed this in the latest commit that can be viewed here: [7a3e7a7 ](https://github.com/kostebudinoski/SunCalcNet/pull/7/commits/7a3e7a7ad176225f72a8487a7cee67625e49564d)